### PR TITLE
Add extra env variables

### PR DESCRIPTION
**Pull request by somanath21**
## Description

* Added extra-env variables 
* Updated extra-env in ph-ee-engine Values.yaml

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/ph-ee-engine/ams-paygops/templates/deployment.yaml [hunk 1]
```diff
@@ -55,6 +55,11 @@ spec:
               value: "{{ .Values.paygops.base_url }}"
             - name: "LOGGING_LEVEL_ROOT"
               value: "{{ .Values.LOGGING_LEVEL_ROOT }}"
+{{- if .Values.extraEnvs | default .Values.deployment.extraEnvs }}
+{{ toYaml ( .Values.extraEnvs | default .Values.deployment.extraEnvs ) | indent 12 }}
+{{- end }}        
+          envFrom: {{ toYaml ( .Values.envFrom | default .Values.deployment.envFrom ) | nindent 12 }}
```
Comment (fynmanoj) [inline helm/ph-ee-engine/ams-paygops/templates/deployment.yaml:61]: the code is not tested. there is a typo in every copy pasted line : 'nindent 12'
what is the use of envFrom ?
please recheck the indentation for `envFrom` and `securityContex`

**Review by fynmanoj — CHANGES_REQUESTED** [review]
the code is not tested

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Please test the code

File: helm/ph-ee-engine/ams-paygops/templates/deployment.yaml (hunk 1 above)
Comment (somanath21) [reply helm/ph-ee-engine/ams-paygops/templates/deployment.yaml:61]: - That isn't typo in extraEnv's indentation.  [ 'nindent' is same as the 'indent' function, it prepends a new line to the beginning of the string ] 
- envFrom reads the values/variables from secret or configmap.
- Indentation for envFrom and securityContext is correct - Checked

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**